src/builtins.c

```diff
@@ -931,22 +931,27 @@ JL_CALLABLE(jl_f__call_in_world_total)
 
 // tuples ---------------------------------------------------------------------
 
-JL_CALLABLE(jl_f_tuple)
+static jl_value_t *arg_tuple(jl_value_t *a1, jl_value_t **args, size_t nargs)
```

Comment:
It seems weird to have two functions that do the same thing, but one breaks in the `nargs==0` case.

Reply:
The difference is that this takes the first argument separately, so it cannot be called in the nargs==0 case, while the purpose of the other function is to peel off that first argument and pass it to this function

Reply:
Ah, got it.